### PR TITLE
Condensing the manuscript

### PR DESCRIPTION
**Pull request by mlap**
A lot of deletions, and blatant cut and pasting into appendix A here. Haven't put too much effort into flow yet, just want to make sure that dropping all of this stuff doesn't diminish coherency.

### CODE REVIEW

File: manuscript/manuscript.Rmd [hunk 1]
```diff
@@ -193,11 +174,9 @@ $$
   \pi^* = \underset{\pi}{\text{argmax}} \,\, \mathbb{E}_{\tau \sim p_\pi(\tau)}\Big[\sum_{t=0}^{H} \gamma^t r(s_t, a_t) \Big] =  \underset{\pi}{\text{argmax}} \,\, J (\pi)
 $$
 
-Note that this formalism extends to an infinite horizon case when $H= \infty$.
-While there are RL methods to deal with infinite horizon cases, we will only present methods for finite horizon or episodic POMDPs.
-In the appendix, we will go into further detail on the general approaches that model-free deep RL algorithms take towards optimizing the RL objective. 
+In Appendix B, we will go into greater detail on the general approaches that deep RL algorithms take towards optimizing the RL objective.
```
[the remark this answers is not in the document]
Comment (mlap) [reply manuscript/manuscript.Rmd:177]: Yeah this is a typo

**Comment by mlap** [comment]
@cboettig Made changes to the RL section in the latest commit. Kept the model-free bit in and a briefer paragraph on neural nets.

**Comment by cboettig** [comment]
This is looking really good. Can we just add one sentence after the bit on model-free and neural nets, that "in this paper, we will focus on model-free approaches which use neural networks"

still on the fence with you about what to call the 'formal concepts' section.  I think it might more accurately be titled: "Markov decision process problems" or something like that?

**Comment by mlap** [comment]
I made some changes to the discussion in the latest commit, bringing us under 5000 words now.

**Comment by cboettig** [comment]
@mlap looks like a few merge conflicts vs the references branch I just merged? :point_up:

**Comment by cboettig** [comment]
I think the changes are good.  Yeah, tough call about Millie's paragraph at the end.  I think we should add a sentence in the middle of what becomes the last paragraph saying something like: 

> a greater role of algorithms in conservation decision-making also raises questions about ethics and power, particularly when those algorithms are opaque or proprietary [@scoville, @chapman] 

(scoville: https://doi.org/10.1016/j.cosust.2021.01.009, chapman: https://doi.org/10.32942/osf.io/jhbuz)

**Comment by mlap** [comment]
> This is looking really good. Can we just add one sentence after the bit on model-free and neural nets, that "in this paper, we will focus on model-free approaches which use neural networks"
> 
> still on the fence with you about what to call the 'formal concepts' section. I think it might more accurately be titled: "Markov decision process problems" or something like that?

@cboettig Probably time to merge this unless you have anything else you want to see done immediately. Re title of this section, its hard for me to decide on something because it ends on paragraphs that are more algorithm-focused than (po)mdp.

**Comment by mlap** [comment]
> > This is looking really good. Can we just add one sentence after the bit on model-free and neural nets, that "in this paper, we will focus on model-free approaches which use neural networks"
> > still on the fence with you about what to call the 'formal concepts' section. I think it might more accurately be titled: "Markov decision process problems" or something like that?
> 
> @cboettig Probably time to merge this unless you have anything else you want to see done immediately. Re title of this section, its hard for me to decide on something because it ends on paragraphs that are more algorithm-focused than (po)mdp-centric.

Reinforcement Learning Problem Statement and Methods?

**Comment by cboettig** [comment]
> Re title of this section, its hard for me to decide on something because it ends on paragraphs that are more algorithm-focused than (po)mdp-centric.

agree, though I think perhaps we ought to move last two paragraphs of the section (neural nets & model free) up to the previous section anyway? They don't depend on the formal definitions, and might be a bit buried by putting them here.

**Pull request merged**